scipy/stats/_axis_nan_policy.py

```diff
@@ -409,17 +439,15 @@ def axis_nan_policy_wrapper(*args, _no_deco=False, **kwds):
 
                 if sentinel:
                     samples = _remove_sentinel(samples, paired, sentinel)
-                return hypotest_fun_in(*samples, **kwds)
+                return hypotest_fun_out(*samples, **kwds)
 
             # check for empty input
             # ideally, move this to the top, but some existing functions raise
             # exceptions for empty input, so overriding it would break
             # backward compatibility.
             empty_output = _check_empty_inputs(samples, axis)
             if empty_output is not None:
-                statistic = empty_output
-                pvalue = empty_output.copy()
-                return result_object(statistic, pvalue)
+                return result_object(*([empty_output]*n_outputs))
```

Comment:
This will just return multiple references to the same object `empty_output`. Previously, it copied `empty_output` and returned separate objects as output. Is this change intended?

Reply:
Not intended. That's a good call. I'm trying to think of credible ways to mutate an empty array in which this would cause problems. Are there any, since there are no elements to be changed?